tests/arrays_test_generated.h

```diff
@@ -0,0 +1,82 @@
+// automatically generated by the FlatBuffers compiler, do not modify
+
+#ifndef FLATBUFFERS_GENERATED_ARRAYSTEST_ARRAYSTEST_TEST1_H_
+#define FLATBUFFERS_GENERATED_ARRAYSTEST_ARRAYSTEST_TEST1_H_
+
+#include "flatbuffers/flatbuffers.h"
+
+namespace ArraysTest {
+namespace Test1 {
+
+struct ArrayStruct;
+
+struct ArrayTable;
+
+MANUALLY_ALIGNED_STRUCT(4) ArrayStruct FLATBUFFERS_FINAL_CLASS {
+ private:
+  float a_;
+  int32_t b_[15];
+  int8_t c_;
+  int8_t __padding0;
+  int16_t __padding1;
+
+ public:
+  ArrayStruct() { memset(this, 0, sizeof(ArrayStruct)); }
+  ArrayStruct(const ArrayStruct &_o) { memcpy(this, &_o, sizeof(ArrayStruct)); }
+  ArrayStruct(float _a, const int32_t *_b, int8_t _c)
+    : a_(flatbuffers::EndianScalar(_a)), c_(flatbuffers::EndianScalar(_c)), __padding0(0), __padding1(0) { memcpy(b_, _b, 60); (void)__padding0; (void)__padding1; }
+
+  float a() const { return flatbuffers::EndianScalar(a_); }
+  void mutate_a(float _a) { flatbuffers::WriteScalar(&a_, _a); }
+  const int32_t *b() const { return b_; }
```

Comment:
note that data is stored in little endian format in structs so this is not great. We should provide a function with an index. If we want to provide a pointer accessor, we should maybe call it `b_LE` or something to make the risk more obvious.